dataschool/_people/blake.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Blake Barnhill
+image: /assets/images/people/blake.png
```

Comment:
Looks good but I think you didn't commit/add your image to the repo?!

Reply:
I believe the image is already in the repo. All of the contributor images were pulled over, just not the bios that went along with them.

Reply:
Atleast the version I forked has all of the contributor images already in the Assets folder.